.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish to (Test)PyPI
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to upload to pypi'
+        required: true
+      pypi_repo:
+        description: 'Repo to upload to ("testpypi" or "pypi")'
+        default: 'testpypi'
+        required: true
+
+jobs:
+  publish:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install -U pip
+        python -m pip install -U setuptools wheel twine
+
+    - name: Publish package to TestPyPI
+      uses: pypa/gh-action-pypi-publish@v1.5.0
```

Comment:
I just noticed this action only publishes the built packages. We need to build them before uploading them.

Reply:
I should have checked, my bad. I added a build step above.